packages/babel-generator/src/generators/base.ts

```diff
@@ -8,7 +10,7 @@ export function File(node: Object) {
   this.print(node.program, node);
 }
 
-export function Program(node: Object) {
+export function Program(this: Printer, node: any) {
```

Comment:
going to update all of `node.any` to use ast node types from @babel/types

Reply:
done - added some `ts-expect-error` todos, and opened separate PR with fixes for AST node definitions in @babel/types